### ISSUE TITLE
Microphone icon appears near wingpanel sound icon

### ISSUE THREAD

**Issue by celiopy** [sodaliterocks/sodalite#98]
_It should, **if**_ I have a microphone, and any app is using it.
The issue is, I have no mic, and the icon appears when any sound is output, a video playing, system (alerts) sounds, anything.

On elementary it doesn't happen. What is going on?

**Comment by electricduck** [comment]
![image](https://github.com/sodaliterocks/sodalite/assets/1594644/f5053ef9-91fd-441c-b7f9-95d2b1a99555)

Is __anything__ appearing under **System Settings ➔ Sound ➔ Input**?

**Comment by celiopy** [comment]
No. I noticed the same behavior using Fedora, with Terra packages, manually installing pantheon.
